ETC.tsv

```diff
@@ -5022,7 +5022,7 @@ ETC_20150317_005021	{@st41}Open your inventory by pressing the {img F2 40 40} ke
 ETC_20150317_005022	{memo X}The Mausoleum lanterns are being attacked by the guardians!
 ETC_20150317_005023	{memo X}Bearkaras' suppressed power has returned.
 ETC_20150317_005024	Panto_Rsword: Hot Panto Fighter: 50/Panto_Rspear: Hot Panto Spearman: 50/Panto_Rhand: Hot Panto: 50/Panto_Rstaff: Hot Panto Wizard: 51
-ETC_20150317_005025	$Crafting the item
+ETC_20150317_005025	$Crafting
```

Comment:
Always seems silly when it comes up, since it's kind of redundant.  What else would you be crafting, but the item?